src/Administration/Resources/app/administration/src/app/component/base/sw-simple-search-field/sw-simple-search-field.scss

```diff
@@ -12,6 +12,7 @@
         .mt-block-field__block {
             border: none;
             box-shadow: none;
+            min-height: none;
```

Comment:
Not sure, if this should be done in the meteor component, or if here is the correct place.

Reply:
If you want you can also add 
```
.sw-block-field__block {
     height: auto;
}
```

to 
`src/Administration/Resources/app/administration/src/module/sw-product/view/sw-product-detail-context-prices/sw-product-detail-context-prices.scss` like this:

<img width="436" alt="Bildschirmfoto 2025-04-29 um 16 32 11" src="https://github.com/user-attachments/assets/3094a74d-813e-449b-9c13-dd2822862020" />

Then we fixed also the Advanced Pricing toolbar:


<img width="890" alt="Bildschirmfoto 2025-04-29 um 16 32 37" src="https://github.com/user-attachments/assets/2a5fb8c5-cebf-4eaf-a4da-44634286a9f5" />

Reply:
Thank you, I will check it. Somehow I though I did `min-height: 0;`. But somehow I pushed the wrong changes.

I will check that this also works for the advanced prices.

Reply:
Perfect, thanks 🙂 👍